src/staking-v3/components/DiscoverV3.vue

```diff
@@ -3,23 +3,37 @@
     <!-- Todo: Delete -->
     <!-- <staking-home /> -->
     <feature-dapp />
-    <staking />
+    <!-- <staking /> -->
```

Comment:
I think we should have Staking component on main dApp staking page. It is weird for me that I can't see my rewards and staked dApps when I come do dApp staking home page. It is fine to have Staking component on Assets page too. cc @Kahonnohak